sw/vendor/riscv_compliance.vendor.hjson

```diff
@@ -13,8 +14,17 @@
   exclude_from_upstream: [
     "doc",
     "spec",
-    "riscv-target",
     "riscv-ovpsim",
+    "riscv-target/Codasip-simulator",
```

Comment:
originally i removed riscv-target, and hoped that the patching would bring it back for the opentitan directory. That didn't work, so I manually called out each sub-directory.